wbSort.pas

```diff
@@ -44,20 +44,20 @@ function wbMergeSortInternal(_A, _B: PwbPointerArray; _Count : Integer; _Compare
   Temp           = EspOffset - SizeOfPointer * 3;
 
   Count          = EspOffset - SizeOfPointer * 4;
-  SegmentSize    = EspOffset - SizeOfPointer * 5;
```

Comment:
this was all changed automatically by github in making my change.  I'm assuming it's all just a whitespace change from tabs to spaces, or something else harmless like that.